files/zh-cn/games/techniques/control_mechanisms/mobile_touch/index.md

```diff
@@ -25,91 +27,91 @@ el.addEventListener("touchcancel", handleCancel);
 
-![Touch controls for the player's ship, with visible output of the x and y position.](controls-touch.png)
+![玩家飞船的触摸控制器，可显示 x 坐标和 y 坐标。](controls-touch.png)
 
-To see if it's working correctly we can output the `x` and `y` positions using the `output` element. The `preventDefault()` function is needed to prevent the browser from moving — without it you'd have the default behaviour, and the Canvas would be dragged around the page, which would show the browser scroll bars and look messy.
+要查看工作是否正常，我们可以使用 `output` 元素输出 `x` 和 `y` 位置。需要使用 `preventDefault()` 函数来防止浏览器移动，否则就会出现默认行为，画布会在页面上被拖动，从而显示浏览器滚动条，看起来乱糟糟的。
 
-## Touch events in Phaser
+## Phaser 中的触摸事件
 
-We don't have to do this on our own; frameworks like Phaser offer systems for managing touch events for us — see [managing the touch events](http://phaser.io/docs/2.6.1/Phaser.Touch.html).
+我们不必自己做这件事；Phaser 等框架为我们提供了管理触摸事件的系统——请参阅[管理触摸事件](https://phaser.io/docs/2.6.1/Phaser.Touch.html)。
 
-### Pointer theory
+### 指针理论
 
-A [pointer](http://phaser.io/docs/2.6.1/Phaser.Pointer.html) represents a single finger on the touch screen. Phaser starts two pointers by default, so two fingers can perform an action at once. Captain Rogers is a simple game — it can be controlled by two fingers, the left one moving the ship and the right one controlling the ship's gun. There's no multitouch or gestures — everything is handled by single pointer inputs.
+[指针](https://phaser.io/docs/2.6.1/Phaser.Pointer.html) 代表触摸屏上的一根手指。Phaser 默认启动两个指针，因此两个手指可以同时执行一个动作。Captain Rogers 是一款简单的游戏——只需用两根手指控制，左手指移动飞船，右手指控制飞船上的火炮。游戏中没有多点触控或手势，所有操作均由单个指针输入完成。
 
-You can add more pointers to the game by using; `this.game.input.addPointer` up to ten pointers can be managed simultaneously. The most recently used pointer is available in the `this.game.input.activePointer` object — the most recent finger active on the screen.
+可以使用 `this.game.input.addPointer` 为游戏添加更多指针，最多可同时管理 10 个指针。最近使用的指针可在 `this.game.input.activePointer` 对象中找到，即屏幕上最近使用的手指。
 
-If you need to access a specific pointer, they are all available at, `this.game.input.pointer1`, `this.game.input.pointer2`, etc. They are assigned dynamically, so if you put three fingers on the screen, then, `pointer1`, `pointer2`, and `pointer3` will be active. Removing the second finger, for example, won't affect the other two, and setting it back again will use the first available property, so `pointer2` will be used again.
+如果需要访问特定指针，它们都可以在 `this.game.input.pointer1`、`this.game.input.pointer2` 等处使用。它们是动态分配的，所以如果你在屏幕上放了三个手指，`pointer1`、`pointer2` 和 `pointer3` 就会处于活动状态。例如，移除第二根手指不会影响其他两根手指，再次设置时将使用第一个可用属性，因此 `pointer2` 将再次被使用。
 
-You can quickly get the coordinates of the most recently active pointer via the `this.game.input.x` and `this.game.input.y` variables.
+你可以通过 `this.game.input.x` 和 `this.game.input.y` 变量快速获取最近激活指针的坐标。
 
-### Input events
+### Input 事件
```

Comment:
maybe the title could be translated, too

Reply:
```suggestion
### 输入事件
```